gap/perm.gi

```diff
@@ -113,7 +113,11 @@ FindHomMethodsPerm.Imprimitive :=
   end;
 
 #! @BeginChunk PcgsForBlocks
-#! TODO
+#! This method is called after a hint is set in
+#! <C>FindHomMethodsPerm.Imprimitive</C>. Therefore the group <A>G</A> preserves
```

Comment:
I am torn here... I want to say that you could use `<Ref Subsect="Imprimitive" Style="Text"/>` here instead -- but then for somebody reading the code, it would be beneficial to see `FindHomMethodsPerm.Imprimitive`. Hrm. Perhaps we can find a way to allow using the label `FindHomMethodsPerm.Imprimitive`, then we could have the advantages of both.

For now, it is perfectly fine to leave this as-is.

Reply:
I changed the `BalTreeForBlocks` acording to the fact that you should also be able to read and understand it in the code. So it is now in kind of the same state as this bit.